vue-frontend/src/components/jobs/FooterSection.vue

```diff
@@ -0,0 +1,206 @@
+<template>
+  <footer class="bottom-footer-div">
+    <div class="division-alignment">
+      <a
+        href="https://www.glassdoor.co.in/Overview/Working-at-Canopas-EI_IE3194462.11,18.htm"
+      >
+        <img
+          src="@/assets/images/footer/glassdoor.webp"
+          class="glassdoor-img"
+        />
+      </a>
+    </div>
+
+    <div class="follow-text">
+      <h4 class="normal-text">Follow us on</h4>
+    </div>
+
+    <!-- font awesome -->
+    <div class="social-media-icons division-alignment">
+      <li v-for="(socialMedia, index) in socialMediaIcons" :key="socialMedia">
+        <a :href="socialMedia.url" class="fab-icon" target="_blank">
+          <font-awesome-icon
+            class="fab"
+            :icon="socialMedia.icon"
+            @mouseover="hoverIndex = index"
+            @mouseleave="hoverIndex = null"
+            :class="{ 'fa-icon': hoverIndex == index }"
+          />
+        </a>
+      </li>
+    </div>
+
+    <div class="text-center">
+      <!-- font awesome -->
+
+      <!-- Copyright -->
+      <p class="copyright-text">
+        <font-awesome-icon :icon="copyrightIcon" />
+        <span id="currentYear"></span> Canopas Software LLP. All rights
+        reserved.
+      </p>
+      <!-- Copyright -->
+    </div>
+  </footer>
+</template>
+<script type="module">
+import { FontAwesomeIcon } from "@fortawesome/vue-fontawesome";
+import {
+  faMedium,
+  faFacebook,
+  faTwitter,
+  faInstagram,
+  faLinkedin,
+} from "@fortawesome/free-brands-svg-icons";
+
+import { faCopyright } from "@fortawesome/free-regular-svg-icons";
+
+import footerLogoImage from "@/assets/images/logo/logo-footer.svg";
+import Config from "@/config.js";
+
+export default {
+  data() {
+    return {
+      copyrightIcon: faCopyright,
+      footerLogoImage: footerLogoImage,
+      hoverIndex: null,
+      socialMediaIcons: [
+        {
+          url: Config.FACEBOOK_URL,
+          icon: faFacebook,
+        },
+        {
+          url: Config.INSTAGRAM_URL,
+          icon: faInstagram,
+        },
+        {
+          url: Config.TWITTER_URL,
+          icon: faTwitter,
+        },
+        {
+          url: Config.MEDIUM_URL,
+          icon: faMedium,
+        },
+
+        {
+          url: Config.LINKEDIN_URL,
+          icon: faLinkedin,
+        },
+      ],
+    };
+  },
+  components: {
+    FontAwesomeIcon,
+  },
+};
+</script>
+
+
+
+<style lang="scss"  scoped>
+.normal-text {
+  font-weight: bold;
+  font-size: 28px;
+  line-height: 34px;
+  color: #3d3d3dcc;
+  margin-bottom: 15%;
+}
+.copyright-text {
+  color: #3d3d3d 80%;
+  font-size: 15px;
+}
+
+.fa-icon {
+  color: white;
+}
+
+.bottom-footer-div {
+  padding: 3% 0;
+  display: flex;
+  flex-direction: column;
+}
+
+.social-media-icons li {
+  margin: 1.6rem 0.5rem 0.6rem 0;
+  display: inline-flex;
+}
+
+.social-media-icons li:hover {
```

Comment:
Make social-media-icon as gradient-btn. On hover, it will behave like `Let's talk business` button of header